docs/decisions/2024-07-11_include_file_limits.md

```diff
@@ -1,4 +1,4 @@
-# 2024-03-26 Appropriate limit for include files per project
+# 2024-07-11 Appropriate limit for include files per project
```

Comment:
I noticed that the date in the header and the date in the file name was different than the date of the ADR, I can undo this in this ticket but thought might as well shape it up